pkg/api/api.go

```diff
@@ -11,8 +11,10 @@ const (
 
 // consts to maintain existing names of various sub-resources
 const (
-	OpenShiftConsoleName      = "openshift-console"
-	OpenShiftConsoleShortName = "console"
-	OpenShiftConsoleNamespace = "openshift-console"
-	OAuthClientName           = OpenShiftConsoleName
+	OpenShiftConsoleName              = "openshift-console"
```

Comment:
Nit: If these all ought to match, should they just all `= OpenShiftConsoleName`?

Reply:
Was thinking about it but the semantics between name and namespace are a bit different. But agreeing on that :)

Reply:
Yeah, I hear ya.  It just happens that the strings match.